sys-libs/libatomic_ops/libatomic_ops-7.6.2.recipe

```diff
@@ -0,0 +1,94 @@
+SUMMARY="An atomic memory update operations portable implementation"
+DESCRIPTION="Semi-portable access to hardware-provided atomic memory update \
+operations on a number architectures."
+HOMEPAGE="https://github.com/ivmai/libatomic_ops"
+COPYRIGHT="1991-1994 Xerox Corporation
+	1996-1999 Silicon Graphics
+	2003-2011 Hewlett-Packard Development Company, L.P.
+	2005, 2007 Thiemo Seufer
+	2007 NEC LE-IT
+	2016 Marek Vasut"
+LICENSE="MIT
+	Boehm
```

Comment:
Hmm, I just noticed that the contents of this file are not the Boehm license. It just mentions that license.
If we keep a `Boehm` in `LICENSE` then this file will be in `/system/data/licenses/Boehm`, potentially hiding a correct copy of the real Boehm license.
So, since that file is the same as `doc/LICENSING.txt`, I'd suggest that we remove `Boehm` from the `LICENSE` variable and, instead, only keep one copy of it, which is already in `/system/documentation/packages/libatomic_ops/LICENSING.txt` when the package is installed.

```diff
 LICENSE="MIT
-	Boehm
	GNU GPL v2"
```
@alaviss, would you be OK with this?

(Sorry for not having noticed this earlier.)
(Everything else is OK :)

Reply:
The LICENSE field should try to be correct. So if some license is used in the code, it should be mentionned here. And of course that means the license file should be there. It can be a symlink if said license is already included in the package, however.

So:

    ln -s $documentationDir/LICENSING.txt $dataDir/licenses/Boehm